desktop/apps/consignments2/components/thank_you/index.js

```diff
@@ -0,0 +1,73 @@
+import Document from '../../../../components/main_layout/public/icons/consignments-doc.svg'
+import GreenCheck from '../../../../components/main_layout/public/icons/green-check.svg'
+import OpposingArrows from '../../../../components/main_layout/public/icons/consignments-opposing-arrows.svg'
+import PropTypes from 'prop-types'
+import React from 'react'
+import SpeechBubble from '../../../../components/main_layout/public/icons/consignments-speech-bubble.svg'
+import { connect } from 'react-redux'
+
+function ThankYou ({ submission }) {
+  const submissionImage = submission.image_url ? submission.image_url : '/images/missing_image.png'
+
+  return (
+    <div className='consignments2-submission-thank-you'>
```

Comment:
I think it is maybe time to add one. Do you have a preference? Otherwise would probably just use `bem-cn` like in prediction.